### PR TITLE
fix: Refactor image interface and update to move llama cloud -> together provider

### PR DESCRIPTION
**Pull request by lalalune**
This PR integrates https://github.com/ai16z/eliza/pull/752 and also updates to move TOGETHER to not be LLAMA_CLOUD since we have lots of llama clouds now :)

### CODE REVIEW

File: packages/plugin-image-generation/src/index.ts [hunk 1]
```diff
@@ -116,9 +126,27 @@ const imageGeneration: Action = {
         const images = await generateImage(
             {
                 prompt: imagePrompt,
-                width: 1024,
-                height: 1024,
-                count: 1,
+                width: options.width || 1024,
+                height: options.height || 1024,
+                ...(options.count !== undefined
+                    ? { count: options.count || 1 }
```
Comment (monilpat) [inline packages/plugin-image-generation/src/index.ts:132]: != null for these as we probably want the {} on null and undefined.

File: packages/core/src/models.ts [hunk 2]
```diff
@@ -107,6 +107,7 @@ export const models: Models = {
             [ModelClass.EMBEDDING]: "llama-3.1-8b-instant",
         },
     },
+    // TODO: Remove LLAMACLOUD and just use Together
```
Comment (monilpat) [inline packages/core/src/models.ts:110]: Should we create an issue for this so it doesn't get lost? Thanks so much!

**Review by monilpat — COMMENTED** [review]
Thanks so much for getting this out! Great work! :)

**Review by monilpat — APPROVED** [review]
LGTM! If there is a way to update the test plan with a quick test run or screen grab would be amazing otherwise good to go :)

File: packages/plugin-image-generation/src/index.ts (hunk 1 above)
Comment (lalalune) [reply packages/plugin-image-generation/src/index.ts:132]: Will change to `...(options.count != null ? { count: options.count || 1 } : {}),`

**Pull request merged**